### ISSUE TITLE
Refactor Tools data for design-systems.md

### ISSUE THREAD

**Issue by t-will-gillis** [hackforla/website#4802]
### Dependency
  - [x] #4860 
  - [x] #4861 
 
### Prerequisites
  1. You must be a member of Hack for LA to work on an issue. If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page.
  2. Please make sure you have read our Hack for LA [Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md) before you claim/start working on an issue.

### Overview
We need to revise the filter menu on the [projects-check](https://www.hackforla.org/projects-check) page to include a dropdown for "Tools", so that we can see where we might need to move project values from `tools` to `technologies`, and vice versa. In order for the filter to work properly, we need to refactor the `tools` data in the .md file from a string to a list.
### Action Items
  - [x] Refer to the [design-systems.md](https://github.com/hackforla/website/blob/gh-pages/_projects/design-systems.md) file and find the entry for `tools:`
  - [x] Change the `tools` data from:
```
tools: Figma, Google Docs, Zoom, Otter.ai
```
to
```
tools:
  - Figma
  - Google Docs
  - Zoom
  - Otter.ai
```
  - [x] Confirm that the Tools data on all project pages is rendered correctly
### Resources/Instructions
  - [Homepage](https://www.hackforla.org)
  - [Projects](https://www.hackforla.org/projects/) page
  - This issue is part of an epic: #4777

**Comment by davidwiese** [comment]
Availability for this week: Today 7/26/23 2-4p PST & 7/27/23 10a-12p PST
My estimated ETA for completing this issue: Thu 7/27/23 12p PST

**Comment by davidwiese** [comment]
Provide Update
1. Progress - I have made the requested changes to design-systems.md
2. Blockers - I'm not sure if these changes are meant to be visibly different yet. I was unable to see a Tools filter in the Docker instance when I saved. The page appears unchanged. Perhaps this is intended until further work is done elsewhere? 
3. Availability - Available tomorrow (7/27/23) in the morning PST. Going out of town Friday-Sunday. I will be back on Monday.
4. ETA - Unsure if complete or not already.
